README.md

```diff
@@ -219,6 +219,7 @@ Please take a look at the [Contribution Guidelines](https://github.com/meilisear
 
 ### Community Blog Posts
 * [Meilisearch: A Minimalist Full-Text Search Engine](https://tech.marksblogg.com/meilisearch-full-text-search.html)
+* :cn: [Vuepress全文搜索终极版-algolia的开源实现meilisearch全接入指南](https://wiki.eryajf.net/pages/dfc792/) - Vuepress full-text search ultimate version-algolia's open source implementation meilisearch full access guide
```

Comment:
Hi @eryajf thanks for your contribution.
I must say that I find it hard to understand the title/description. We don't think ourselves as an implementation of Algolia, but as an alternative. So I find it quite confusing.
Would you ke ok to change it somehow? Thanks

Reply:
done